components/sub-components/SelectedArticle.jsx

```diff
@@ -0,0 +1,66 @@
+import { IoChatbubbleOutline } from "react-icons/io5";
+import { FaRegHeart } from "react-icons/fa";
+import styles from "./SelectedArticle.module.css";
+import { lineSpinner } from "ldrs";
+import { useState, useEffect } from "react";
+import { fetchArticleById } from "../../utils/api.js";
+import { Link } from "react-router-dom";
+
+lineSpinner.register();
+
+const SelectedArticle = ({article_id, votes, loading, setLoading}) => {
+  
+  const [selectedArticle, setSelectedArticle] = useState([]);
+  
+  useEffect(() => {
+    setLoading(true);
+    fetchArticleById(article_id).then((fetchedArticle) => {
+      setLoading(false);
+      setSelectedArticle(fetchedArticle);
+    });
+  }, []);
```

Comment:
One thing to double check...

Your API should be returning an `object` here, but your state is being set as an `array` on line 13 so it should probably be initialised as an empty object instead `{}`